### PR TITLE
Add senslogo's pin

### PR DESCRIPTION
**Pull request by senslogo**
Edit to already existing pull request **practice**

### CODE REVIEW

**Comment by Brnjamino4school** [comment]
YOU CAN DO ITS(I AM BEING HELD HOSTAGE FORCED TO  DO THIS)

File: _pins/senslogo.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: senslogo
+latitude:45.377910
+Longitude:-75.660106
+---
```
Comment (hectorsector) [inline _pins/senslogo.json]: @senslogo just add a space between the colon `:` and the number and that should allow the tests to pass

**Comment by brianamarie** [comment]
Hi @senslogo! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@senslogo  I'm going to merge now. Nice work!

**Pull request merged**